contrib/debian/netdata.logrotate

```diff
@@ -0,0 +1,13 @@
+/var/log/netdata/*.log {
```

Comment:
this has nothing to do with debian, please submit as separate patch into system/